lib/core/src/Cardano/Wallet.hs

```diff
@@ -753,17 +752,18 @@ newWalletLayer tracer bp db nw tl = do
         -> Maybe UTCTime
         -> Maybe SortOrder
         -> ExceptT ErrListTransactions IO [TransactionInfo]
-    _listTransactions wid mStart mEnd _mOrder = do
+    _listTransactions wid mStart mEnd mOrder = do
         (w, _) <- withExceptT ErrListTransactionsNoSuchWallet $ _readWallet wid
         case (mStart, mEnd) of
             (Just start, Just end) -> when (start > end) $ throwE $
                 ErrListTransactionsStartTimeLaterThanEndTime $
                     ErrStartTimeLaterThanEndTime start end
             _ -> pure ()
         let tip = currentTip w ^. #slotId
+        let order = maybe Descending id mOrder
```

Comment:
Maybe the walletLayer should take a `SortOrder` instead of a `Maybe SortOrder` and leave the `Api` to decide on the default.

Reply:
I do agree :+1: